### PR TITLE
In 138164685 add phone numbers to screening people

### PR DESCRIPTION
**Pull request by sivaOSI**

### CODE REVIEW

File: spec/features/screening/participant/editing_participant_spec.rb [hunk 1]
```diff
@@ -142,6 +150,80 @@
...
+        fill_in 'Phone Number', with: '789-456-1245'
+      end
+
+      marge.phone_numbers.first.number = '789-456-1245'
+
+      stub_request(:put, api_participant_path(marge.id))
+        .with(body: as_json_without_root_id(marge))
+        .and_return(status: 200,
+                    body: marge.to_json,
+                    headers: { 'Content-Type' => 'application/json' })
```
Comment (brunoOSI) [inline spec/features/screening/participant/editing_participant_spec.rb]: you can use the json_body helper for this like so
`json_body(marge.to_json, status: 200)`

File: spec/features/screening/participant/editing_participant_spec.rb [hunk 2]
```diff
@@ -142,6 +150,80 @@
...
+      end
+      expect(
+        a_request(:put, api_participant_path(marge.id))
+        .with(json_body(as_json_without_root_id(marge)))
+      ).to have_been_made
+    end
+
+    within show_participant_card_selector(marge.id) do
+      within '.card-body' do
+        expect(page).to have_selector("#phone-number-#{marge.phone_numbers.first.id}")
```
Comment (brunoOSI) [inline spec/features/screening/participant/editing_participant_spec.rb]: usually we just do <#phone-number-id> not <#phone-number-#id>

File: spec/features/screening/participant/editing_participant_spec.rb [hunk 3]
```diff
@@ -142,6 +150,80 @@
...
+      end
+    end
+
+    marge.phone_numbers = []
+
+    stub_request(:put, api_participant_path(marge.id))
+      .with(body: as_json_without_root_id(marge))
+      .and_return(status: 200,
+                  body: marge.to_json,
+                  headers: { 'Content-Type' => 'application/json' })
```
Comment (brunoOSI) [inline spec/features/screening/participant/editing_participant_spec.rb]: you can use the json_body helper for this like so
`json_body(marge.to_json, status: 200)`

File: app/assets/javascripts/components/people/PhoneNumbersEditView.jsx [hunk 4]
```diff
@@ -32,9 +32,9 @@ export default class PhoneNumbersEditView extends React.Component {
       <div id='phone-numbers'>
         {
           phoneNumbers.map((numbers, index) => {
-            const {number, type} = numbers.toJS()
+            const {id, number, type} = numbers.toJS()
             return (
-              <div key={index} className='row list-item'>
+              <div key={index} className='row list-item' id={`number-${id}`}>
```
Comment (brunoOSI) [inline app/assets/javascripts/components/people/PhoneNumbersEditView.jsx]: is this id used by anything? id's should be unique for a page

File: spec/features/screening/participant/editing_participant_spec.rb (hunk 2 above)
Comment (sivaOSI) [inline spec/features/screening/participant/editing_participant_spec.rb]: The "extra" hash symbol is for interpolation. The result is `phone-number-99` (where 99 is the ID).

**Comment by NateOSI** [comment]
looks good

**Pull request closed without merging**